.github/workflows/cmake.yml

```diff
@@ -54,6 +54,14 @@ jobs:
             codecov: ubuntu_gcc_osb
             cflags: -O3
 
+          - name: Ubuntu GCC -O3 OSB add_subdirectory
+            os: ubuntu-latest
+            compiler: gcc
+            cxx-compiler: g++
+            build-dir: ../build
+            build-src-dir: ../zlib-ng/test/add-subdirectory-project
+            cflags: -O3
```

Comment:
I don't think we need `cflag` or `-O3` in the name.

Reply:
Done